code/test/Fakes/Solution/FakeSolution.cs

```diff
@@ -91,6 +91,18 @@ public class FakeSolution
 		{0}.Release|x86.Build.0 = Release|Any CPU
 	";
 
+        private const string WpfProjectConfigurationTemplate = @"		{0}.Debug|Any CPU.ActiveCfg = Debug|x86
```

Comment:
based on the rename in line 72, this could do with a more descriptive name too

Reply:
done